grouper/fe/templates/permission.html

```diff
@@ -26,35 +31,41 @@
 {% block content %}
 <div class="row">
     <div class="col-md-10 col-md-offset-1">
-        {% if permission.audited %}
-        <div class="alert alert-warning">
-            <strong>This is an audited permission.</strong> Any group this permission is applied
-            to will become audited.
-        </div>
-        {% endif %}
-
-        <table class="table table-elist">
-            <thead>
-                <tr>
-                    <th class="col-sm-2">Group</th>
-                    <th class="col-sm-5">Argument</th>
-                </tr>
-            </thead>
-            <tbody>
-            {% for mapped_group in mapped_groups %}
-                <tr>
-                    <td><a href="/groups/{{mapped_group.groupname}}">{{mapped_group.groupname}}</a></td>
-                    <td>{{mapped_group.argument|default("(unargumented)", True)}}</td>
-                </tr>
-            {% endfor %}
-            {% if not mapped_groups %}
-                <tr>
-                    <td colspan="2" class="text-center"><em>This permission is not mapped to any
-                    groups. To grant this permission to a group, visit the group page.</em></td>
-                </tr>
+        {% if not permission.enabled %}
+            <div class="alert alert-warning">
+                <strong>This permission is disabled.</strong> It continues to exist for the record but otherwise has no effect.
+            </div>
+        {% else %}
+            {% if permission.audited %}
+                <div class="alert alert-warning">
+                    <strong>This is an audited permission.</strong> Any group this permission is applied
+                    to will become audited.
+                </div>
             {% endif %}
-            </tbody>
-        </table>
+
```

Comment:
this is just indenting the whole block into the else block of "if permission is disabled"